### PR TITLE
fix(bridge-ui): dropdown css and color css fix

### PR DESCRIPTION
**Pull request by ukurenov**
You use one dropdown component with two different pages, on Faucet page it not will be fixed width.
I added new boolean props isWidthFull, that you can use it from faucet component for dropdown will be 100% width
Fix example:
**Before:**
image

**After:**
image

Color fix in mobile version light style in disabled button:hover (Was been white on white) and change css cursor pointer and not-allowed(it was been in wrong place and didn't work)
I can't send hover screen, you can test it here:
Screenshot 2024-05-05 at 11 09 46 PM

Transactions page -- Select source Chain

### CODE REVIEW

File: packages/bridge-ui/src/components/Faucet/Faucet.svelte [hunk 1]
```diff
@@ -148,7 +148,7 @@
         direction={ChainSelectorDirection.SOURCE}
         label={$t('chain_selector.currently_on')}
         switchWallet />
-      <TokenDropdown {disabled} tokens={mintableTokens} {onlyMintable} bind:value={selectedToken} />
```
Comment (KorbinianK) [inline packages/bridge-ui/src/components/Faucet/Faucet.svelte:151]: remove this again, it is not meant to be fullscreen.

File: packages/bridge-ui/src/components/TokenDropdown/DropdownView.svelte [hunk 2]
```diff
@@ -26,11 +26,13 @@
   export let value: Maybe<Token> = null;
   export let selectToken: (token: Token) => void = noop;
   export let onlyMintable: boolean = false;
+  export let isWidthFull: boolean = false;
```
Comment (KorbinianK) [inline packages/bridge-ui/src/components/TokenDropdown/DropdownView.svelte:29]: remove

File: packages/bridge-ui/src/components/TokenDropdown/DropdownView.svelte [hunk 3]
```diff
@@ -26,11 +26,13 @@
...
 
   let addArc20ModalOpen = false;
 
   $: menuClasses = classNames(
     'menu absolute right-0 w-[265px] p-3 mt-2 rounded-[10px] bg-neutral-background z-10  box-shadow-small',
+    isWidthFull ? 'w-full' : undefined,
```
Comment (KorbinianK) [inline packages/bridge-ui/src/components/TokenDropdown/DropdownView.svelte:35]: remove

File: packages/bridge-ui/src/components/TokenDropdown/TokenDropdown.svelte [hunk 4]
```diff
@@ -228,6 +229,7 @@
 
   {#if isDesktopOrLarger}
     <DropdownView
+      {isWidthFull}
```
Comment (KorbinianK) [inline packages/bridge-ui/src/components/TokenDropdown/TokenDropdown.svelte:232]: remove

**Review by KorbinianK — CHANGES_REQUESTED** [review]

**Review by KorbinianK — APPROVED** [review]

**Review by dantaik — APPROVED** [review]

**Pull request merged**